applications/StructuralMechanicsApplication/python_scripts/structural_response.py

```diff
@@ -129,6 +164,12 @@ def __init__(self, identifier, response_settings, model_part):
         self.model_part = model_part
         self.model_part.AddNodalSolutionStepVariable(SHAPE_SENSITIVITY)
 
+    def RunCalculationProcess(self, calculate_gradient):
+        self.Initialize()
```

Comment:
Here we should use the base implementation calling all the functions (even if some of them may just pass).